_posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: 'New WildFly S2I and Runtime Multi-arch Images'
+date: 2022-12-05
+tags: wildfly galleon cloud wildfly-s2i openshift
+description: This article provides details on the new S2I and runtime multi-arch images
+author: jfdenise
+---
+
+This article provides details on the new S2I and runtime multi-arch images.
+
+== New WildFly S2I and Runtime Multi-arch images
+
+These new multi-arch images (`linux/arm64` in addition to `linux/amd64`) have a different naming scheme than the current WildFly images 
+to better handle multiple JDK versions and align with the tag scheme used for the WildFly centos7 docker images 
+(as explained in https://www.wildfly.org/news/2022/11/10/wildfly-docker-temurin/[this] blog post).
```

Comment:
This is just a personal opinion, others might disagree, but I found using the title of the post "Updates on WildFly Docker Images" more comfortable to read than 'this'. Perhaps even personalize with 'as explained in Jeff Mesnil's https://....[Updates on WildFly Docker Images] blog post.'